python/servo/command_base.py

```diff
@@ -832,6 +832,10 @@ def run_cargo_build_like_command(
         if with_debug_assertions or self.config["build"]["debug-assertions"]:
             env['RUSTFLAGS'] = env.get('RUSTFLAGS', "") + " -C debug_assertions"
 
+        # mozjs gets its Python from `env['PYTHON3']`, which defaults to `python3`,
+        # but uv venv on Windows only provides a `python`, not `python3`.
+        env['PYTHON3'] = "python"
```

Comment:
Could we only apply this to windows? On some of my Linux systems `python` does not exist, so this would probably cause issues

Reply:
the uv venv always provides `python` on Linux :)

Reply:
Interesting, that does seem to be the case also on my machine without a system `python`. Do you happen to know if this is documented somewhere? I couldn't find any documentation on it, but perhaps I was searching for the wrong keywords.

Reply:
I think that is how standard python virtual environments work
https://docs.python.org/3/library/venv.html#how-venvs-work

> This will prepend that directory to your PATH, so that running python will invoke the environment’s Python interpreter and you can run installed scripts without having to use their full path.

Reply:
Interesting, that's definitely good to know for the future!